packages/web/src/components/modal-radio/ModalRadioItem.module.css

```diff
@@ -27,6 +27,10 @@
   width: 24px;
 }
 
+.root:has(.input:disabled) {
```

Comment:
i personally don't like doing nested selectors, since there is always a chance that the inner-workings of a component can change. it looks like the pattern has already been set down below, so this is fine, but if we have access to the disabled prop in he root, i might add a classname just for the root?

Reply:
yea i was using the pattern that was set here already, but i dont love it either. will fix